### PR TITLE
refactor(calendar): remove ie11 BC year workaround

### PR DESCRIPTION
**Pull request by Nantawat-Poothong**
## Description
IE11 series: remove specific IE11 code from `ef-calendar`

Fixes # (issue)
https://jira.refinitiv.com/browse/STG-33

## Type of change

- [x] Refactor (improves code without changing its functionality)

### CODE REVIEW

File: packages/elements/src/calendar/index.ts [hunk 1]
```diff
@@ -1030,16 +1025,9 @@ export class Calendar extends ControlElement implements MultiValue {
    */
   private viewFormattedDate (segment: DateSegment, includeMonth = false): TemplateResult {
     const year = segment.year;
-    const isBC = year <= 0;
-    const includeEra = isBC;
+    const includeEra = year <= 0;
```
Comment (wsuwt) [inline packages/elements/src/calendar/index.ts]: I feel that previous code is a bit more clear when specifying `isBC`. What do you think ?

Comment (Nantawat-Poothong) [reply packages/elements/src/calendar/index.ts]: I think the TranslateDirective will receive the variable name `includeEra` anyway.

how about this?
```suggestion
    const isBC = year <= 0;
```

  and return  with option `{ date, includeMonth, includeEra: isBC }`

Comment (wsuwt) [reply packages/elements/src/calendar/index.ts]: I think it's better than before. Looks good.

**Comment by Sarin-Udompanish** [comment]
Should we remove `formatLocaleDate` method from the code ?

**Comment by wsuwt** [comment]
Are we able to remove `case: 'Esc'` as well ? Otherwise LGTM.

**Comment by Nantawat-Poothong** [comment]
> Are we able to remove `case: 'Esc'` as well ? Otherwise LGTM.

Ah, I didn't see it, thanks for reminding me.

**Review by Sarin-Udompanish — APPROVED** [review]

**Pull request merged**